documentation/index.md

```diff
@@ -0,0 +1,639 @@
+
+### Functionality showcased
+
+- Place Search
+- Geocode addresses
+- Reverse Geocode
+- Turn-by-turn Directions
+- Dynamically switch basemaps
+- Open Web Maps
+- Work with ArcGIS Online or an on-premise ArcGIS Portal
+- OAuth2 authentication
+
+Configure this app for your organization or use it to learn how to integrate similar capabilities into your own app.
+
+## Overview
+
+*Disclaimer: For screen real estate purposes, only Android and WPF screenshots are posted. Since the app was built with Xamarin Forms, iOS and UWP user interfaces are very similar to the Android interface, with exception of the iconography and platform specific controls.*
+
+When the app starts, you will be presented with the default empty map. In the initial run of the app, you may be prompted to accept that the app wants to use your current location. 
+
+| Android | WPF |
+|:-------:|:---:|
+|<img src="/documentation/images/start_screen_and.png" width="450"/>|<img src="/documentation/images/start_screen_wpf.png"/>|
+
+The default map is created inside the `MapViewModel.cs` by initializing the `Map` with a new Topographic Vector basemap. 
+
+```csharp
+private Map _map = new Map(Basemap.CreateTopographicVector());
+
+/// <summary>
+/// Gets or sets the map
+/// </summary>
+public Map Map
+{
+    get
+    {
+        return _map;
+    }
+
+    set
+    {
+        _map = value;
+        OnPropertyChanged();
+    }
+}
+```
+
+Inside the view (XAML), a `MapView` control is created and its `Map` property is bound to the `Map` property on the `MapViewModel`
+
+**Xamarin & WPF:**
+
+```xml
+<esri:MapView x:Name="MapView" Map="{Binding Map, Source={StaticResource MapViewModel}}"/> 
+```
+
+## Place search & geocoding
+
+At the top of the screen, there is a menu button and a search bar. The search bar provides the geocoding functionality. [Geocoding](https://developers.arcgis.com/net/latest/wpf/guide/search-for-places-geocoding-.htm) lets you transform an address or a place name to a specific geographic location. Reverse geocoding lets you use a geographic location to find a description of the location, like a postal address or place name. 
+
+In the solution, the logic for geocoding is contained inside the shared `GeocodeViewModel`. First, a `LocatorTask` is defined to use the [Esri World Geocoder](https://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer). Before using the `LocatorTask`, it must be loaded. The loadable pattern is described [here](https://developers.arcgis.com/net/latest/wpf/guide/loadable-pattern.htm).
```

Comment:
[Esri World Geocoder] links to a 403 and is not very helpful. Can we swap this link for something else? I'm not sure which I like more:

- https://developers.arcgis.com/features/geocoding/
- https://developers.arcgis.com/rest/geocode/api-reference/overview-world-geocoding-service.htm

Thoughts?

Reply:
I'm guessing the original intention in this doc was to reference the service used by the app (see below), but I've replaced it with the feature overview page so that it links to something that has useful, readable info.

https://github.com/Esri/maps-app-dotnet/blob/8f65f7e92239df7dae248c55244a16c08c1f9970/src/MapsApp.Shared/Helpers/Configuration.cs#L31